integrationTests/vm/txsFee/multiShard/asyncCall_test.go

```diff
@@ -122,8 +122,8 @@ func TestAsyncCallShouldWork(t *testing.T) {
 	scr = intermediateTxs[0]
 	utils.ProcessSCRResult(t, testContextSecondContract, scr, vmcommon.Ok, nil)
 
-	require.Equal(t, big.NewInt(49993150), testContextSecondContract.TxFeeHandler.GetAccumulatedFees())
-	require.Equal(t, big.NewInt(4999315), testContextSecondContract.TxFeeHandler.GetDeveloperFees())
+	require.Equal(t, big.NewInt(3680), testContextSecondContract.TxFeeHandler.GetAccumulatedFees())
```

Comment:
why these changes?
If these changes are required because the activation flag is set for the transferAndAsyncCallbackFixEnableEpoch, then, I think we should double the tests, one with the active flag and one with the disabled flag in order to easily assure backwards compatibility.

Reply:
because the consumption was wrong. it was completely wrong for different reasons - like 2 fixes in process.go. Backward compatibility is ensured by flags.

Reply:
We can duplicate one of these tests if needed.

Reply:
ok, got it